command/exec_test.go

```diff
@@ -31,7 +31,7 @@ func TestExecCommand_implements(t *testing.T) {
 func TestExecCommandRun(t *testing.T) {
 	t.Parallel()
 	cfg := agent.TestConfig()
-	cfg.DisableRemoteExec = agent.Bool(false)
+	cfg.DisableRemoteExec = &agent.BoolFalse
```

Comment:
I'll revert to the `Bool()` function since it is more flexible in situations where you need to derive the pointer from another value. The main issue is the use of `*bool` in the first place IMO.